### PR TITLE
Added default values for Role

### PR DESCRIPTION
**Pull request by ptambellini01**

### CODE REVIEW

File: Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/SamlConstants.java [hunk 1]
```diff
@@ -100,6 +100,8 @@
     public static final String USER_ORG_ID_ATTR = "urn:oasis:names:tc:xspa:1.0:subject:organization-id";
     public static final String HOME_COM_ID_ATTR = "urn:nhin:names:saml:homeCommunityId";
     public static final String USER_ROLE_ATTR = "urn:oasis:names:tc:xacml:2.0:subject:role";
+    public static final String USER_SYST_ATT = "2.16.840.1.113883.6.96";
+    public static final String USER_SYST_NAME_ATT = "SNOMED_CT";
```
Comment (jsmith2) [inline Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/SamlConstants.java]: Can you follow the same naming conventions as the other variables, namely change "ATT" to "ATTR"?

Comment (ptambellini01) [reply Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/SamlConstants.java]: Ok

File: Product/Production/Services/PolicyEngineCore/src/test/java/gov/hhs/fha/nhinc/policyengine/adapter/orchestrator/util/PolicyEngineUtilTest.java [hunk 2]
```diff
@@ -72,21 +73,21 @@ public void testNoOrgId() {
     @Test
     public void testNoRoleCodeSystemName() {
         AssertionType assertion = getFullAssertion();
-        
+
         assertion.getUserInfo().getRoleCoded().setCodeSystemName(null);
         PolicyEngineUtil util = new PolicyEngineUtil();
         CheckPolicyResponseType resp = util.checkAssertionAttributeStatement(assertion);
-        assertDeny(resp);
+        assertPermit(resp);
```
Comment (jsmith2) [inline Product/Production/Services/PolicyEngineCore/src/test/java/gov/hhs/fha/nhinc/policyengine/adapter/orchestrator/util/PolicyEngineUtilTest.java]: I think this unit test can just be removed.

**Comment by jsmith2** [comment]
PR looks good.  Please just update the variable name, removing the junit test is optional (just don't think it tests anything)

**Review by jsmith2 — APPROVED** [review]

**Review by TabassumJafri — APPROVED** [review]
Approving without any comments

**Pull request merged**